package-lock.json

```diff
@@ -0,0 +1,3593 @@
+{
```

Comment:
The project is built with Yarn, not npm, so please build accordingly and remove `package-lock.json`

Reply:
@Juilzze resolved this issue in latest commit.